src/api/ootdUpload.ts

```
@@ -1,22 +1,18 @@
-import {
-  getPresignedUrls,
-  uploadToPresignedUrl,
-  imageSourceToBlob,
-} from "@/api/presignedUrl";
+import {getPresignedUrls, uploadToPresignedUrl, imageSourceToBlob,} from "@/api/presignedUrl";
```

ESLint

[error] Replace `getPresignedUrls,·uploadToPresignedUrl,·imageSourceToBlob,` with `·getPresignedUrls,·uploadToPresignedUrl,·imageSourceToBlob·`

(prettier/prettier)